src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -37,6 +37,11 @@ class DefaultStorageKeyAllocator
 
     const char * FabricTable(chip::FabricIndex fabric) { return Format("f/%x/t", fabric); }
 
+    // Access Control List
+
+    const char * AccessControlList() { return Format("acl"); }
+    const char * AccessControlEntry(size_t index) { return Format("acl/%zx", index); }
```

Comment:
This will not work on some platforms due to missing "%z" specifier. Suggest:

const char * AccessControlEntry(size_t index) { return Format("acl/%x", static_cast<unsigned>(index)); }

Reply:
I've created https://github.com/project-chip/connectedhomeip/pull/14547 to fix this.